doc/lua_api.txt

```diff
@@ -7069,7 +7069,8 @@ Used by `minetest.register_node`, `minetest.register_craftitem`, and
     {
         description = "Steel Axe",
         -- Can contain new lines. "\n" has to be used as new line character.
-        -- Defaults to the item's name.
+        -- If not given or an empty string, the item's name is used instead
+        -- in some places.
```

Comment:
wait, where is "some places"?
if it's just `get_description()` the extra fallback should be documented there

Reply:
`get_description()` is just the lua api function. `ItemStack::getDescription` uses the name as default, so "some places" are places where the engine shows / uses / gets the description.

Reply:
In case it's not clear: I support adding that to the documentation and please do so.

Reply:
Done.
(It was not clear.)